documentcloud/documents/admin.py

```diff
@@ -56,3 +58,25 @@ def save_model(self, request, obj, form, change):
 
     def delete_model(self, request, obj):
         obj.destroy()
+
+
+@admin.register(Entity)
+class EntityAdmin(admin.ModelAdmin):
+    """Entity Admin"""
+
+    list_display = ("name", "kind", "mid", "wikipedia_url")
+    list_filter = ("kind")
```

Comment:
This needs to be a list or tuple.... one of python's downfalls is that `("string") == "string"` and you need to do `("string",)` with the trailing comma for a tuple of size 1

Reply:
Whoa, very good to know.